python/pager/.gitignore

```diff
@@ -0,0 +1,5 @@
+*~
```

Comment:
The values should just be in the top-level `.gitignore`

Reply:
Ok, will fix that and the pyc of course

Reply:
Done as requested.